packages/validator/src/lib/PwaValidator.ts

```diff
@@ -80,6 +93,26 @@ export class PwaValidator {
     const performancePass = performanceScore >= MIN_PERFORMANCE_SCORE;
     const passed = pwaPass && performancePass;
     const accessibilityScore = psiResult.lighthouseResult.categories.accessibility.score;
+
+    // Web Vitals Scores
+    const fcpScore =
+      psiResult.lighthouseResult.audits.metrics.details.items[0]['firstContentfulPaint'];
+    const fcpStatus = 'PASS';
+
+    const lcpScore =
+      psiResult.lighthouseResult.audits.metrics.details.items[0]['largestContentfulPaint'];
+    const lcpStatus =
+        this.getStatus(Math.round(lcpScore / 100) * 100, MIN_LCP_PASS_SCORE, MIN_LCP_WARN_SCORE);
```

Comment:
What are we doing here? Are we trying to make the output number end with two zeros (eg `1234` -> `1200`)?

Reply:
Yes.

Reply:
OK - that's a little bit non-obvious at first glance - could you add some nicely named convenience method?

Reply:
Moved this to a function.